rock-paper-scissors/service/src/main.rs

```diff
@@ -0,0 +1,218 @@
+use oasis_std::{Address, Context};
+#[macro_use]
+extern crate serde;
+
+#[derive(oasis_std::Service)]
+struct RockPaperScissors {
+    player_name: String,
+    player_move: Move, // TODO would be better as an enum?
+    challenger_name: String,
+    challenger_move: Move,
+    played: bool, // whether or not someone can challenge. true = player has played. false = nobody has played or just finished.
```

Comment:
```suggestion
	/// Whether or not someone can challenge. `true` means that the player has played.
    played: bool, 
```

Reply:
but you don't actually need this value since `player_move.is_some()` once the player has played